packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_row.dart

```diff
@@ -25,18 +29,45 @@ class LoggingTableRow extends StatefulWidget {
 
   final bool isSelected;
 
-  static TextStyle get metadataStyle =>
-      Theme.of(navigatorKey.currentContext!).subtleTextStyle;
+  static TextStyle get metadataStyle {
+    final currentContext = navigatorKey.currentContext;
+    assert(
+      currentContext != null,
+      'LoggingTableRow.metadataStyle requires a valid navigatorKey to be set. If this assertion is hit in tests then make sure to `wrap()` the widget being pumped.',
+    );
+    return Theme.of(navigatorKey.currentContext!).subtleTextStyle;
```

Comment:
nit: use `Theme.of(currentContext)` since we have already looked it up and stored it in a local